docs/DeveloperGuide.md

```diff
@@ -119,17 +126,24 @@ within each transaction and how the transactions are associated with the `Transa
 The `TransactionList` holds a dynamic array list that can store multiple `Transaction` objects.
 
 Each Transaction object contains the following mandatory member attributes:
+
 1. **category** A category for the transaction.
 2. **description** More information regarding the transaction, written without any space.
 3. **amount** Value of the transaction in numerical form. Only integers within 0 and 10000000 is accepted.
 4. **date** Date of the transaction. The format must be in "yyyyMMdd".
 
 Some important operations are performed within the `TransactionList` class, which implements the following:
-- `TransactionList#addIncome(String description, int amount, String category, LocalDate date)` - Adds a transaction of class type Income into the transactions list.
```

Comment:
Similarly, there are unexpected line breaks over here, please pull the changes from the repository and remove these line breaks, to avoid overwriting parts belonging to other team members. Thanks!

Reply:
Think these is due to the reformatting done by the compiler settings.

Reply:
I see, then in that way it should not be able to be altered. No worries!